script.js

```diff
@@ -0,0 +1,216 @@
+
+  if (player === 'rock' && computer === 'rock') {
+    changeComRock();
+    return 'DRAW';
+  }
+
+  if (player === 'rock' && computer === 'scissors') {
+    changeComScissors();
+    playerScore.textContent = `${(playerGoal += 1)}`;
+    return 'YOU WIN';
+  }
+  if (player === 'rock' && computer === 'paper') {
+    changeComPaper();
+    computerScore.textContent = `${(computerGoal += 1)}`;
+    return 'YOU LOSE';
+  }
+
+  return 'odd call 😬';
+};
+
+const playRoundForScissors = () => {
+  const player = gameBodyScissorsTextPlayer.textContent;
+  const computer = gameBodyScissorsTextComputer.textContent;
+
+  if (player === 'scissors' && computer === 'scissors') {
+    changeComScissors();
+    return 'DRAW';
+  }
+
+  if (player === 'scissors' && computer === 'paper') {
+    playerScore.textContent = `${(playerGoal += 1)}`;
+    changeComPaper();
+    return 'YOU WIN';
+  }
+  if (player === 'scissors' && computer === 'rock') {
+    changeComRock();
+    computerScore.textContent = `${(computerGoal += 1)}`;
+    return 'YOU LOSE';
+  }
+
+  return 'odd call 😬';
+};
+
+const getPlayerRock = () => {
+  gameBodyRockTextPlayer.textContent = 'rock';
+  gameBodyRockTextComputer.textContent = getComputerChoice();
+  const playerSvgRock = document.querySelector('.player-option-svg-player path');
+
+  playerSvgRock.setAttribute('d', dPathRock);
+  gameBodyBigCirclePlayerOptionBorder.setAttribute('class', 'change-rock-circle-border');
+
+  gameBodyDiv.style.display = 'none';
+  afterPlayerComputerChoiceDiv.style.display = 'block';
+  winLoseText.textContent = playRoundForRock();
+};
+
+const getPlayerScissors = () => {
+  gameBodyScissorsTextPlayer.textContent = 'scissors';
+  gameBodyScissorsTextComputer.textContent = getComputerChoice();
+  const playerSvgRock = document.querySelector('.player-option-svg-player path');
+
+  playerSvgRock.setAttribute('d', dPathScissors);
+  gameBodyBigCirclePlayerOptionBorder.setAttribute('class', 'change-scissors-circle-border');
+
+  gameBodyDiv.style.display = 'none';
+  afterPlayerComputerChoiceDiv.style.display = 'block';
+  winLoseText.textContent = playRoundForScissors();
+};
+
+const getPlayerPaper = () => {
+  gameBodyPaperTextPlayer.textContent = 'paper';
+  gameBodyPaperTextComputer.textContent = getComputerChoice();
+  const playerSvgRock = document.querySelector('.player-option-svg-player path');
+
+  playerSvgRock.setAttribute('d', dPathPaper);
+  gameBodyBigCirclePlayerOptionBorder.setAttribute('class', 'change-paper-circle-border');
+
+  gameBodyDiv.style.display = 'none';
+  afterPlayerComputerChoiceDiv.style.display = 'block';
+
+  winLoseText.textContent = playRoundForPaper();
+};
+
+const playAgain = () => {
+  gameBodyDiv.style.display = 'flex';
+  afterPlayerComputerChoiceDiv.style.display = 'none';
+};
+
+const closeRulesBtnFunc = () => {
+  closeRulesBtn.style.display = 'none';
+};
+
+const openRulesBtnFunc = () => {
+  openRulesBtn.style.display = 'block';
```

Comment:
Look at how they use classList to replace the displaying and not displaying in the link that I send you up in the first one